CHANGELOG.md

```diff
@@ -75,6 +75,9 @@ These changes are available on the `master` branch, but have not yet been releas
   ([#2206](https://github.com/Pycord-Development/pycord/pull/2206))
 - Added function `Guild.delete_auto_moderation_rule`.
   ([#2153](https://github.com/Pycord-Development/pycord/pull/2153))
+- Added
+  [`USE_CLYDE_AI` permission](https://github.com/discord/discord-api-docs/pull/6354).
```

Comment:
What about the role subscription purchase notifications flag?

Reply:
^ + the changelog entry is entirely wrong